uaclient/config.py

```diff
@@ -828,10 +836,33 @@ def simulate_status(
             }
         )
 
+        now = datetime.now(timezone.utc)
         if contract_info.get("effectiveTo"):
             response["expires"] = contract_info.get("effectiveTo")
+            expiration_datetime = util.parse_rfc3339_date(response["expires"])
```

Comment:
TIL we aren't using DatetimeAwareJsonDecoder in util.readurl! We probably should so that we don't need to parse later on like here, but that's probably a separate PR

Reply:
Let's create a card/issue for this one - must be done with care